### PR TITLE
feat(cli): improve README.md for application projects

### PR DESCRIPTION
**Pull request by raymondfeng**
1. Fixes https://github.com/strongloop/loopback-next/issues/6379
2. Improve the README.md for generated applications

Signed-off-by: Raymond Feng <enjoyjava@gmail.com>

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [x] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/cli/generators/app/templates/README.md.ejs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## Run the application
+
+```sh
+npm start
+```
+
+You can also run `node .` to skip the build step.
+
+Open http://127.0.0.1:3000 in your browser.
```
Comment (emonddr) [inline packages/cli/generators/app/templates/README.md.ejs:29]: Do we want to mention the other http format?

```
http://[::1]:3000
```

File: packages/cli/generators/app/templates/README.md.ejs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Rebuild the project
+
+To incrementally build the project:
```
Comment (emonddr) [inline packages/cli/generators/app/templates/README.md.ejs:33]: does it really only do it incrementally? Or does it rebuild everything?

File: packages/cli/generators/app/templates/README.md.ejs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+## Other useful commands
+
+- `npm run migrate`: Migrate database schemas for models
+- `npm run openapi-spec`: Generate OpenAPI spec into a file
+- `npm run docker:build`: Build a Docker image for this application
+- `npm run docker:run`: Run this application inside a Docker container
+
+## Tests
```
Comment (emonddr) [inline packages/cli/generators/app/templates/README.md.ejs:68]: is this npm test verbose or not?  I personally don't like seeing non-verbose '..........' when tests are running. CI is fine.
I like seeing verbose checkmarks etc. Do we want to also mention?
```
npm test -- -R spec
```

**Review by emonddr — CHANGES_REQUESTED** [review]

File: packages/cli/generators/app/templates/README.md.ejs (hunk 1 above)
Comment (raymondfeng) [reply packages/cli/generators/app/templates/README.md.ejs:29]: No, it's safer and simpler to use `127.0.0.1`.

File: packages/cli/generators/app/templates/README.md.ejs (hunk 2 above)
Comment (raymondfeng) [reply packages/cli/generators/app/templates/README.md.ejs:33]: It's incremental for `tsc` with `tsconfig.tsbuildinfo` cache.

File: packages/cli/generators/app/templates/README.md.ejs (hunk 3 above)
Comment (raymondfeng) [reply packages/cli/generators/app/templates/README.md.ejs:68]: No, that's too advanced for README.

File: packages/cli/generators/app/templates/README.md.ejs (hunk 1 above)
Comment (emonddr) [reply packages/cli/generators/app/templates/README.md.ejs:29]: ok. It is the default http format i see when I start an lb4 app though.

**Review by emonddr — APPROVED** [review]
thx, Raymond. LGTM

**Pull request merged**